.drone.yml

```diff
@@ -335,3 +335,54 @@ steps:
     instance:
       - drone-publish.rancher.io
 
+---
+kind: pipeline
+name: harvester-plugin
+
+depends_on:
+- test
+
+trigger:
+  ref:
+  - "refs/heads/release-harvester-plugin-v*"
+  - "refs/heads/master"
+  event:
+  - push
+  - tag
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: build
+  pull: default
+  image: node:14
+  commands:
+  - ./scripts/build-hosted
+
+- name: build-pkg
+  pull: default
+  image: node:14
+  commands:
+  - ./shell/scripts/build-pkg.sh harvester
+
+- name: upload-gate
+  pull: default
+  image: node:14
+  commands:
+  - ./scripts/build-upload-gate
+
+- name: upload
+  pull: default
+  image: plugins/gcs
+  environment:
+    PKG_VERSION: ${DRONE_PKG_VERSION}.tar.gz
```

Comment:
Can this also upload the un-packaged files as well? This would support the UI loading output directly from `releases.rancher.com/harvester-ui` aka
```
https://releases.rancher.com/harvester-ui/plugin/harvester-1.0.x-head/harvester-1.0.x-head.umd.min.js
and
https://releases.rancher.com/harvester-ui/plugin/harvester-1.0.x/harvester-1.0.x.umd.min.js
```

The dashboard build process would still pull in the packaged version (to support the dashboard embedded use case)

Reply:
The results of my tests were as expected.   It correctly places the folder and tar.gz file into the 'reviews.rancher.com/harvester-ui/plugin' directory.

Reply:
Ah, maybe i misunderstood. To confirm the folder contains 

`releases.rancher.com/harvester-ui/plugin/harvester-1.0.x/harvester-1.0.x.umd.min.js`
`releases.rancher.com/harvester-ui/plugin/harvester-1.0.x/<other package files>`

as well as

`releases.rancher.com/harvester-ui/plugin/harvester-1.0.x/harvester-1.0.x.tar.gz`

?

Reply:
I think  `releases.rancher.com/harvester-ui/plugin/harvester-1.0.x/harvester-1.0.x.tar.gz`     =>  `releases.rancher.com/harvester-ui/plugin/harvester-1.0.x.tar.gz`.

Reply:
We would need the unpacked files of the plugin as well to support https://github.com/rancher/rancher/pull/38831/files#diff-b2f1f4f0c50b7452d729bccde70315eb06368408c5062c97febbe7dd6f87e2edR93 (feature supports testing of a un-released 1.0.3 plugin, so it would be used instead of the 1.0.3 plugin embedded in the dashboard)